src/features/auth/ui/logout/useLogout.tsx

```diff
@@ -0,0 +1,17 @@
+import { useDispatch } from 'react-redux'
```

Comment:
add index.ts in logout 
add index.ts in logoutModal
add export from logout in index.ts in ui